lib/beaker-answers/answers.rb

```diff
@@ -40,6 +40,17 @@ class Answers
       :q_pe_check_for_updates                        => 'n',
     })
 
+    DEFAULT_HIERA_ANSWERS = StringifyHash.new.merge(flatten_keys_to_joined_string({
+      'console_admin_password' => DEFAULT_ANSWERS[:q_puppet_enterpriseconsole_auth_password],
+      'puppet_enterprise' => {
+        'puppetdb_database_password'     => DEFAULT_ANSWERS[:q_puppetdb_password],
+        'classifier_database_password'   => DEFAULT_ANSWERS[:q_classifier_database_password],
+        'activity_database_password'     => DEFAULT_ANSWERS[:q_activity_database_password],
+        'rbac_database_password'         => DEFAULT_ANSWERS[:q_rbac_database_password],
+        'orchestrator_database_password' => DEFAULT_ANSWERS[:q_orchestrator_database_password],
+        'use_application_services'       => true,
```

Comment:
Yeah I was thinking about that but apparently forgot to do it. It'd be good to continue to test that, and I think some of the tests might even be dependent on those names/users. I'll patch that up.